java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -71,4 +71,68 @@ public interface ServerManagementClusterCommands {
      *     value is the information of the sections requested for the node.
      */
     CompletableFuture<ClusterValue<String>> info(InfoOptions options, Route route);
+
+    /**
+     * Rewrites the configuration file with the current configuration.<br>
+     * The command will be routed automatically to all nodes.
+     *
+     * @see <a href="https://redis.io/commands/config-rewrite/">redis.io</a> for details.
+     * @return <code>OK</code> when the configuration was rewritten properly, otherwise an error is
+     *     raised.
+     * @example
+     *     <pre>
```

Comment:
throughout - should there be empty spaces before the `<pre>`?

Reply:
Tricks of java linter (spotless)